scripts/bulk-access-control.md

```diff
@@ -0,0 +1,93 @@
+# Bulk Access Control Management script
+[Back to the list of all defined scripts](./index.md)
+
+<!-- TOC -->
+* [Import script](#bulk-access-control-management-script)
+  * [What it does?](#what-it-does)
+  * [Who can use it?](#who-can-use-it)
+  * [When we use it?](#when-we-use-it)
+  * [How it Works?](#how-it-works)
+  * [Parameters](#parameters)
+  * [Conclusion](#conclusion)
+* [What is an Access Condition?](#what-is-an-access-condition)
+<!-- TOC -->
+## What it does?
+
+This script performs bulk changes to the access conditions of the items and bitstreams. 
+The Bulk Access Control Management focus on Access Control rather than resource policies. The goal will be to add / replace access condition for a set of objects from a user friendly interface that will abstract from the complexity/internal rules of DSpace’s resource policies and bundles as much as possible.
+It takes a json file as input to identify the items and bitstreams target of the changes and the access conditions to append or set.
+
+## Who can use it?
+
+This script can be used by any user that has administrative priviled over at least one community, collection or item limited to the resources that they admin.
```

Comment:
Misspelling: `priviled` should be `privileges`.

Reply:
done